Assets/Scripts/KitchenSerializer.cs

```diff
@@ -27,6 +27,10 @@ private void Start()
         {
             DeserializeKitchen();
         }
+        else
+        {
+            SerializeKitchen(false);
```

Comment:
sets the data object so that a kitchen can run without having been read from a file